cypress/integration/lock-exceptions/user-adds-lock-exception.js

```diff
@@ -35,16 +35,16 @@ When(`the user clicks on the 'Add lock exception' button`, () => {
     cy.findByRole('button', { name: 'Add lock exception' }).click()
 })
 
-Then('a modal appears', () => {
-    cy.getWithDataTest('{add-lock-exception-modal}').should('exist')
+Then('the user is sent to the add lock exception page', () => {
+    cy.url().should('contain', '/lock-exceptions/add')
     cy.wait('@fetchDataSets')
 })
 
 When('the user selects a data set', () => {
-    cy.getWithDataTest('{add-lock-exception-modal-select-data-set}')
-        .find('.d2-ui-selectfield')
-        .click()
-    cy.findByRole('menuitem', { name: 'ART monthly summary' }).click()
+    cy.getWithDataTest('{add-lock-exception-select-data-set}').click()
+    cy.getWithDataTest('{dhis2-uicore-singleselectoption}')
```

Comment:
Unfortunately `cy.findByRole('menuitem', { name: 'ART monthly summary' }).click()` does not work with the `@dhis2/ui` single select option due to missing `role` attributes.